public/javascripts/src/actions/FeedActionCreators.jsx

```diff
@@ -0,0 +1,13 @@
+var AppDispatcher = require('../dispatcher/AppDispatcher.js');
+var LoungeConstants = require('../constants/LoungeConstants.js');
+
+var ActionTypes = LoungeConstants.ActionTypes;
+
+module.exports = {
+
+  loadPosts: function() {
+    AppDispatcher.dispatch({
+      type: ActionTypes.LOAD_POSTS
+    })
+  }
+}
```

Comment:
also, why don't we set module.exports equal to the function over an object with the loadPosts function?